composer.json

```diff
@@ -20,8 +20,8 @@
   },
   "require": {
     "php": ">=7.0",
-    "magento/framework": "^100.1.0",
-    "magento/module-catalog": "^101.0.0",
+    "magento/framework": "^101.0.0",
+    "magento/module-catalog": "^102.0.0",
     "magento/module-catalog-search": "^100.1.0"
```

Comment:
Do we need to bump this up?

Reply:
Nope, M2.2.2 is running magento/module-catalog-search v100.2.0, which is covered by this